### PR TITLE
feat: Adding SDKDefault Authentication

### PR DESCRIPTION
**Pull request by jayendranarumugam**
As part of https://github.com/tomkerkhove/promitor/pull/2578 updating the docs

-Enabling DefaultAzureAuth

### CODE REVIEW

File: docs/resource-discovery/runtime-configuration.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Example:
 
 ```yaml
 authentication:
-  # Options are ServicePrincipal, SystemAssignedManagedIdentity, UserAssignedManagedIdentity.
+  # Options are ServicePrincipal, SystemAssignedManagedIdentity, UserAssignedManagedIdentity, SdkDefault.
```
Comment (tomkerkhove) [inline docs/resource-discovery/runtime-configuration.md:52]: Can you please update section above?

File: docs/security/azure-authentication.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ The app key can also be read from a file by specifying the file's location:
 
 ```yaml
 authentication:
-  # Options are ServicePrincipal, SystemAssignedManagedIdentity, UserAssignedManagedIdentity.
+  # Options are ServicePrincipal, SystemAssignedManagedIdentity, UserAssignedManagedIdentity, SdkDefault.
```
Comment (tomkerkhove) [inline docs/security/azure-authentication.md:40]: Same here, update Supported Authentication Mechanisms

**Review by tomkerkhove — CHANGES_REQUESTED** [review]

File: docs/resource-discovery/runtime-configuration.md (hunk 1 above)
Comment (jayendranarumugam) [reply docs/resource-discovery/runtime-configuration.md:52]: Updated please check now

File: docs/security/azure-authentication.md (hunk 2 above)
Comment (jayendranarumugam) [reply docs/security/azure-authentication.md:40]: Updated

**Comment by tomkerkhove** [comment]
Thank you!

**Pull request merged**